pydruid/db/sqlalchemy.py

```diff
@@ -189,7 +189,7 @@ def get_columns(self, connection, table_name, schema=None, **kwargs):
         return [
             {
                 "name": row.COLUMN_NAME,
-                "type": type_map[row.DATA_TYPE.lower()],
+                "type": jdbc_type_map[row.JDBC_TYPE],
```

Comment:
If we introduce new types in the future this would break again. Do you know if SQLAlchemy can handle "unknown" types? If so, it'd be good to report some kind of "unknown" type instead of throwing an error here.

If it can't handle unknown types, skipping the column is better than throwing an error.

Reply:
Good point, looks like sqlalchemy offers a [NullType](https://docs.sqlalchemy.org/en/14/core/type_api.html#sqlalchemy.types.NullType) for exactly this case.

I added the fallback and also added a warning log similar to other dialects (e.g. [Hive](https://github.com/songyanho/PyHive/blob/1f39ab88d1a1de97e9d98b96894d0c59db34d6a5/pyhive/sqlalchemy_hive.py#L259-L261)).

Reply:
Looks great. Thanks!